### ISSUE TITLE
module API viewer

### ISSUE THREAD

**Issue by ghdk** [ghdk/os#4]

**Comment by ghdk** [comment]
```
make -C /lib/modules/3.16.0-4-586/build M=${PWD} CONFIG_OS_MODULE_P_C=m modules
```

**Comment by ghdk** [comment]
http://stackoverflow.com/questions/16390004/change-default-console-loglevel-during-boot-up

**Comment by ghdk** [comment]
debugging note: virtualbox + linux + 

```
/etc/default/grub
GRUB_CMDLINE_LINUX_DEFAULT="quiet loglevel=7"
```

**Comment by ghdk** [comment]
tristate

**Comment by ghdk** [comment]
Write the section "Module API"

**Comment by ghdk** [comment]
module parameters:
https://www.linux.com/learn/kernel-newbie-corner-everything-you-wanted-know-about-module-parameters

**Comment by ghdk** [comment]
https://www.linuxvoice.com/be-a-kernel-hacker/

**Comment by ghdk** [comment]
Fixed in commit a2094ef6ac7ac57c38a0c967f62370f213b4bead.
